KicadModTree/nodes/specialized/PolygoneLine.py

```diff
@@ -23,17 +23,39 @@
 class PolygoneLine(Node):
     def __init__(self, **kwargs):
         Node.__init__(self)
-        self.polygone_line = kwargs['polygone']
 
-        self.layer = kwargs.get('layer', 'F.SilkS')
-        self.width = kwargs.get('width')
+        self.layer = kwargs.get('layer','F.SilkS')
+        self.width = kwargs.get('width',0.15)
+        
+        self._initMirror(**kwargs)
+        
+        self._initPolygone(**kwargs)
 
         self.virtual_childs = self._createChildNodes(self.polygone_line)
 
+    def _initMirror(self, **kwargs):
+    
+        self.mirror = [None,None]
+        if kwargs.get('x_mirror') and type(kwargs['x_mirror']) in [float,int]:
+            self.mirror[0] = kwargs['x_mirror']
+        if kwargs.get('y_mirror') and type(kwargs['y_mirror']) in [float,int]:
+            self.mirror[1] = kwargs['y_mirror']
+
+    def _initPolygone(self, **kwargs):
+        self.polygone_line = kwargs['polygone']
+        
+        for point in self.polygone_line:
+        
+            if self.mirror[0] is not None:
+                point['x'] = 2 * self.mirror[0] - point['x']
+            if self.mirror[1] is not None:
+                point['y'] = 2 * self.mirror[1] - point['y']
+            
     def _createChildNodes(self, polygone_line):
         nodes = []
 
         for line_start, line_end in zip(polygone_line, polygone_line[1:]):
+        
```

Comment:
I think spaces after function definitions, if, for,... should not be made. Only for structuring logic pieces between parts of a function